src/applications/caregivers/components/FormFields/AddressWithAutofill.jsx

```diff
@@ -23,29 +20,30 @@ const PrimaryAddressWithAutofill = props => {
     idSchema,
     onChange,
     schema,
-    veteranAddress,
   } = props;
   const { reviewMode, submitted } = formContext;
-  const { properties: schemaProps, required: reqFields } = schema;
+  const { properties: schemaProps } = schema;
+  const reqFields = ['street', 'city', 'state', 'postalCode', 'county'];
```

Comment:
With the updated web component schema pattern there is no longer a required fields array passed in. Lets set it manually here.